docs/source/command_line.rst

```diff
@@ -151,6 +151,11 @@ imports.
     all modules. For more information on what the other options do,
     see :ref:`Following imports <follow-imports>`.
 
+.. option:: --no-follow-imports-for-stubs
```

Comment:
It is better to document the flag without `no-` prefix.

Reply:
Fixed.